File: boj7/3003.cc
// 킹, 퀸, 룩, 비숍, 나이트, 폰
// https://www.acmicpc.net/problem/3003

#include <stdio.h>

const int arr[6] = {1, 1, 2, 2, 2, 8};

int main() {
  for (int i = 0; i < 6; i++) {
    int x;
    scanf("%d", &x);
    printf("%d ", arr[i] - x);
  }
  printf("\n");
  return 0;
}

File: past_archive/23boj_py/3003.py
# 킹, 퀸, 룩, 비숍, 나이트, 폰
# https://www.acmicpc.net/problem/3003

import sys

a, b, c, d, e, f = map(int, sys.stdin.readline().split())
print(f'{1 - a} {1 - b} {2 - c} {2 - d} {2 - e} {8 - f}')
